label_studio/io_storages/aperturedb/serializers.py

```diff
@@ -0,0 +1,71 @@
+"""This file and its contents are licensed under the Apache License 2.0. Please see the included NOTICE for copyright information and LICENSE for a copy of the license.
+"""
+import os
+
+from io_storages.aperturedb.models import ApertureDBExportStorage, ApertureDBImportStorage, ApertureDBStorageMixin
+from io_storages.serializers import ExportStorageSerializer, ImportStorageSerializer
+from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+
```

Comment:
This file exports the configuration information?

Reply:
Yes.